components/account/home/UserDevicesTable.vue

```diff
@@ -1,20 +1,23 @@
 <script setup lang="ts">
 import type { DataTableColumn, TablePaginationData } from '@rotki/ui-library';
 import type { UserDevice } from '~/types/account';
-import { set } from '@vueuse/core';
+import { get, set } from '@vueuse/core';
 import { formatDate } from '~/utils/date';
 
 const { t } = useI18n({ useScope: 'global' });
 
+const confirmDialog = ref(false);
+const deviceToDelete = ref<UserDevice | null>(null);
```

Comment:
```suggestion
const deviceToDelete = ref<UserDevice>();
```

is there any reason to initialize it to null?